src/icon-manifest/manifestMerger.ts

```diff
@@ -1,49 +1,39 @@
 import * as _ from 'lodash';
-import {
-  IFileCollection,
-  IFolderCollection,
-  IExtensionCollection,
-  IExtension,
-  FileFormat,
-  IIconSchema,
-  IIconGenerator,
-  IFileDefault,
-  IFolderDefault,
-} from '../models';
+import * as model from '../models';
```

Comment:
I think that I'm exactly doing what we do in `.NET`. Alas, here we don't have `namespaces` to import from.